Templateのsubmitでformatが使えない

Templateのオプションのひとつであるsubmitコマンドがformatを解釈しない問題が発生しています。
具体的には次のような状況にてエラーが発生します。

```zsh
$ acc new abc001; cd abc001/a; tree .
########################################
# atcoder-cli/online-judge-tools outputs
########################################
.
|-- a.py
`-- test
    |-- sample-1.in
    |-- sample-1.out
    |-- sample-2.in
    |-- sample-2.out
    |-- sample-3.in
    `-- sample-3.out

$ acc s
submit to: https://atcoder.jp/contests/abc001/tasks/abc001_1
[INFO] online-judge-tools 11.1.3 (+ online-judge-api-client 10.8.0)
[INFO] problem recognized: AtCoderProblem.from_url('https://atcoder.jp/contests/abc001/tasks/abc001_1'): https://atcoder.jp/contests/abc001/tasks/abc001_1
[ERROR] [Errno 2] No such file or directory: '{tasklabel}.py'
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/onlinejudge_command/main.py", line 280, in main
    run_program(parsed, parser=parser)
  File "/usr/local/lib/python3.8/dist-packages/onlinejudge_command/main.py", line 250, in run_program
    submit(args)
  File "/usr/local/lib/python3.8/dist-packages/onlinejudge_command/subcommand/submit.py", line 42, in submit
    with args.file.open('rb') as fh:
  File "/usr/lib/python3.8/pathlib.py", line 1218, in open
    return io.open(self, mode, buffering, encoding, errors, newline,
  File "/usr/lib/python3.8/pathlib.py", line 1074, in _opener
    return self._accessor.open(self, flags, mode)
FileNotFoundError: [Errno 2] No such file or directory: '{tasklabel}.py'
```

`template.json`や`online-judge-tools`・`atcoder-cli`のバージョンは以下の通りです。

```json
{
  "task": {
    "program": [["main.py", "{tasklabel}.py"]],
    "submit": "{tasklabel}.py"
  }
}

```

```zsh
 $ oj --version; acc --version
online-judge-tools 11.1.3 (+ online-judge-api-client 10.8.0)
2.1.0
```

どこかに間違っている設定があるのでしょうか。

--- a/src/project.ts
+++ b/src/project.ts
@@ -207,7 +207,7 @@ export async function installTask(detailed_task: DetailedTask, dirname: string, 
 	// 新しく情報を付与したTaskオブジェクトを返す
 	const s: any = {directory: {path: dirname}};
 	if (flg_tests) s.directory.testdir = testdir;
-	if (task_template !== undefined && task_template.submit !== undefined) s.directory.submit = task_template.submit;
+	if (task_template !== undefined && task_template.submit !== undefined) s.directory.submit = formatTaskDirname(task_template.submit, task, index, contest);
 	return Object.assign(task, s);
 }
 

